packages/lib/response/cache.ts

```diff
@@ -0,0 +1,50 @@
+import { revalidateTag } from "next/cache";
+
+interface RevalidateResponses {
+  environmentId?: string;
+  personId?: string;
+  responseId?: string;
+  singleUseId?: string;
+  surveyId?: string;
+}
+
+export const responseCache = {
+  tag: {
+    byEnvironmentId(environmentId: string) {
+      return `environments-${environmentId}-responses`;
+    },
+    byResponseId(responseId: string) {
+      return `responses-${responseId}`;
+    },
+    byPersonId(personId: string) {
+      return `person-${personId}-responses`;
+    },
+    bySingleUseId(surveyId: string, singleUseId: string) {
+      return `survey-${surveyId}-singleuse-${singleUseId}-responses`;
```

Comment:
also should be `surveys` instead of `survey`

Reply:
to be consistent I would also like `singleUse` better than `singleuse`

Reply:
Done